Formula/soapysdr.rb

```diff
@@ -0,0 +1,34 @@
+class Soapysdr < Formula
+  desc "Vendor and platform neutral SDR support library"
+  homepage "https://github.com/pothosware/SoapySDR/wiki"
+  url "https://github.com/pothosware/SoapySDR/archive/soapy-sdr-0.8.0.tar.gz"
+  sha256 "cff3b1fb1cd3b88932f37df42f69fc3d8e0ea42f4788b35131ceaf5d1b908ac2"
+  license "BSL-1.0"
+  head "https://github.com/pothosware/SoapySDR.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "python@3.9"
+
+  def install
+    args = []
+    args += ["-DENABLE_PYTHON=OFF"]
+    args += ["-DENABLE_PYTHON3=ON"]
+    args += ["-DCMAKE_MODULE_LINKER_FLAGS_INIT='-undefined dynamic_lookup'"]
+    args += %W[-DSOAPY_SDR_ROOT='#{HOMEBREW_PREFIX}']
+
+    unless build.head?
+      args += ["-DSOAPY_SDR_EXTVER=release"]
+    end
```

Comment:
Can just use:
```suggestion
    args << "-DSOAPY_SDR_EXTVER=release" unless build.head?
```

---

Probably need to go through other CMake args to make sure they are actually needed.
For example, `SOAPY_SDR_ROOT` is set to `CMAKE_INSTALL_PREFIX` by default
https://github.com/pothosware/SoapySDR/blob/soapy-sdr-0.8.0/CMakeLists.txt#L37

And whether `CMAKE_MODULE_LINKER_FLAGS_INIT` is still needed.
They do have similar logic in Python bindings build for macOS
https://github.com/pothosware/SoapySDR/blob/soapy-sdr-0.8.0/python3/CMakeLists.txt#L79
So, want to check if this is still required for build, or if it was a workaround before above line was added.

Afterward, can clean up style to use single assignment to `args` rather than appending with `+=` as most other `cmake`-build formulae look like.